journal-letters/original/06-chapter-4.adoc

```diff
@@ -120,8 +136,12 @@ and I am in hopes we shall be clear of that trouble here.
 
 I remain with love to thyself and family; thy friend,
 
+[.signed-section-signature]
 Moses Brown.
 
+--
+
+[.offset]
```

Comment:
I think this would look better if we just put a small break above and below this chunk about Moses Brown, and use the brackets (like you have them) instead of offsetting each paragraph.